rafthttp/peer.go

```diff
@@ -32,9 +32,8 @@ const (
 	// tcp keepalive failing to detect a bad connection, which is at minutes level.
 	// For long term streaming connections, rafthttp pkg sends application level linkHeartbeat
 	// to keep the connection alive.
-	// For short term pipeline connections, rafthttp MUST kill the connection to avoid it being
+	// For short term pipeline connections, the connection MUST be killed to avoid it being
 	// put back to http pkg connection pool.
```

Comment:
we do it through not keeping any idle connections in transport. could we also mention that to remind ourselves?